packages/mobile/src/web3/actions.ts

```diff
@@ -1,7 +1,8 @@
 export enum Actions {
   SET_ACCOUNT = 'WEB3/SET_ACCOUNT',
   SET_ACCOUNT_IN_WEB3_KEYSTORE = 'WEB3/SET_ACCOUNT_IN_WEB3_KEYSTORE',
-  SET_COMMENT_KEY = 'WEB3/SET_COMMENT_KEY',
+  SET_DATA_ENCRYPTION_KEY = 'WEB3/SET_DATA_ENCRYPTION_KEY',
```

Comment:
Not sure there is a better place to move this, but I find that /web3 is not where I would expect to find the data encryption key-related sagas. Realize this is nice to have in the same place as where we're storing a user's account

Reply:
Yeah I agree. In fact neither should be in web3.

Reply:
I've moved most of the DEK stuff to a new web3/dataEncryptionKey.ts file which keeps it in web3 but avoids cluttering the other web3 files. That's probably a good place to stop for this PR I think